feature-configs/rhacm-manifests/operator-install/dpdk/placement-cluster-cnf21.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-cluster-cnf21
+  namespace: dpdk
+spec:
+  clusterSelector:
+    matchExpressions:
+      - key: name
+        operator: In
+        values:
+        - cnf21
```

Comment:
what's cnf21?
is that just to serve as an example?

Reply:
cnf21 is the name (see the key field in the `placementRule`) of the targetted cluster. Yes, it is an example of `placementRule`, since this object is required to let ACM know which cluster or clusters is/are going to install the CNF components.